src/earn/EarnDepositBottomSheet.tsx

```diff
@@ -224,6 +246,24 @@ export default function EarnDepositBottomSheet({
               />
             </Trans>
           </Text>
+        ) : (
+          <Text style={styles.footer}>
+            <Trans
+              i18nKey="earnFlow.depositBottomSheet.noTermsUrlFooter"
+              tOptions={{ providerName: pool.appName }}
+            >
+              <Text
```

Comment:
Should we conditionally render this if `APP_ID_TO_PROVIDER_DOCUMENTS_URL[pool.appId]` is undefined? Also What happens to the translation in that case?

Reply:
updated in [0cfe241](https://github.com/valora-inc/wallet/pull/6127/commits/0cfe241921fc01295d13438d7c8d96ec77483d9f)